src/main.h

```diff
@@ -21,7 +23,9 @@ class DDWAF : public Napi::ObjectWrap<DDWAF> {
     explicit DDWAF(const Napi::CallbackInfo& info);
 
     // JS instance methods
-    void update(const Napi::CallbackInfo& info);
+    Napi::Value update_config(const Napi::CallbackInfo& info);
+    Napi::Value remove_config(const Napi::CallbackInfo& info);
```

Comment:
We can't return `Napi::Boolean` because when we throw an error we have to return undefined is that it ?
```suggestion
    Napi::Boolean update_config(const Napi::CallbackInfo& info);
    Napi::Boolean remove_config(const Napi::CallbackInfo& info);
```

Reply:
`error: no viable conversion from returned value of type 'Value' to function return type 'Napi::Boolean'`

Reply:
![image](https://github.com/user-attachments/assets/aa71d7ea-c065-47fc-800b-8287c045a6c5)
:laughing: i was saying that we can't do what i'm suggesting, and was asking you to confirm it